webui.py

```diff
@@ -220,7 +224,47 @@ def ip_advance_checked(x):
                                     choices=[flags.desc_type_photo, flags.desc_type_anime],
                                     value=flags.desc_type_photo)
                                 desc_btn = gr.Button(value='Describe this Image into Prompt')
+                                desc_image_size = gr.Markdown(label='Image Size', value='Image Size: None', elem_id='desc_image_size')
                                 gr.HTML('<a href="https://github.com/lllyasviel/Fooocus/discussions/1363" target="_blank">\U0001F4D4 Document</a>')
+                                def trigger_image_sizes(image):
+                                    try:
+                                        if image is not None:
+                                            image = Image.fromarray(np.uint8(image))
+                                            width, height = image.size
+                                            ratio = round(width / height, 2)
+                                            gcd = math.gcd(width, height)
+                                            lcm_ratio = f'{width//gcd}:{height//gcd}'
+                                            size_info = f'Image Size: {width} x {height} , Ratio: {ratio} , {lcm_ratio}'
+
+                                            recommended_sizes = [
+                                                '704*1408', '704*1344', '768*1344', '768*1280', '832*1216', '832*1152',
+                                                '896*1152', '896*1088', '960*1088', '960*1024', '1024*1024', '1024*960',
+                                                '1088*960', '1088*896', '1152*896', '1152*832', '1216*832', '1280*768',
+                                                '1344*768', '1344*704', '1408*704', '1472*704', '1536*640', '1600*640',
+                                                '1664*576', '1728*576'
+                                            ]
```

Comment:
this could be modules.config.default_aspect_ratio or do you explicitly only want to recommend core SDXL resolutions?